src/error.ts

```diff
@@ -1,9 +1,28 @@
+function assembleMessage(errors: [message: string, path: string[]][]) {
```

Comment:
MESSAGES.. assemble!

Reply:
![image](https://user-images.githubusercontent.com/13323507/149909283-14b7562a-8d4c-4585-b7f0-94b610afe7f3.png)

Reply:
That's not even the right scene for the picture omg